src/components/Audioplayer/ScrollButton/index.js

```diff
@@ -1,9 +1,19 @@
 import React, { PropTypes } from 'react';
+import styled from 'styled-components';
 import OverlayTrigger from 'react-bootstrap/lib/OverlayTrigger';
 import Tooltip from 'react-bootstrap/lib/Tooltip';
 import LocaleFormattedMessage from 'components/LocaleFormattedMessage';
 
-const style = require('../style.scss');
+const ScrollLink = styled.a`
+  width: 100%;
+  display: inline-block;
+  cursor: pointer;
+  padding-right: 1.5%;
+  color: ${props => (props.active ? props.theme.brandPrimary : props.theme.textColor)};
+  outline: none;
+  cursor: pointer;
+  margin-bottom: 0;
+`;
```

Comment:
Really good questions @naveed-ahmad ! 

I think the goal to this is to eliminate CSS, JS and HTML from being in three different places (well, HTML and JS are in the same file which is JSX) - this leads to:
1. Repeating CSS
2. Dead CSS
3. Poor organization

What's special about `styled-components` is it will only inject CSS that is needed for rendering. Thus, if you never visit the about page, you never get that CSS. One would say 'we can just separate the CSS by page' but with Sass depending on each other, and shared CSS across different places, it's very difficult. Especially with components. Rather, we can code split by our JS files and only download and render what is needed.

I didn't do an amazing job of grouping common components such as `ControlItem` on purpose. I'd like to suggest our CSS move to `styled-components` then combine commonalities. 

@sabeurthabti thoughts? 

Also, I agree about component libs @sabeurthabti we should do the same.